examples/routing/delegated-routing-client/main_test.go

```diff
@@ -25,15 +25,15 @@ func TestFindProviders(t *testing.T) {
 		if r.URL.Path == "/routing/v1/providers/"+cidStr {
 			w.Header().Set("Content-Type", "application/x-ndjson")
 			w.Write([]byte(`{"Schema":"peer","ID":"12D3KooWM8sovaEGU1bmiWGWAzvs47DEcXKZZTuJnpQyVTkRs2Vn","Addrs":["/ip4/111.222.222.111/tcp/5734"],"Protocols":["transport-bitswap"]}` + "\n"))
-			w.Write([]byte(`{"Schema":"peer","ID":"12D3KooWB6RAWgcmHAP7TGEGK7utV2ZuqSzX1DNjRa97TtJ7139n","Addrs":["/ip4/127.0.0.1/tcp/5734"],"Protocols":["transport-horse"]}` + "\n"))
```

Comment:
Perhaps keep `"transport-horse"` and add an additional unknown one so that it's clear it's getting filtered